src/chopper_pack.cpp

```diff
@@ -41,6 +40,18 @@ int set_up_and_parse_subparser_split(seqan3::argument_parser & parser, pack_conf
     parser.add_option(config.alpha, 'a', "alpha",
                       "The scaling factor to influence the number of merged bins.");
 
+    parser.add_option(config.output_filename, 'o', "outfile",
+                      "An output file name for the binning results.");
+
+    parser.add_option(config.aggregate_by_column, 'y', "aggregate-by",
+                      "Which column do you want to aggregate your files by? Start counting your columns from 1!",
+                      seqan3::option_spec::standard,
+                      seqan3::arithmetic_range_validator{3, std::numeric_limits<int>::max()});
```

Comment:
Btw, does `aggregate_by_column` need to be an int? Couldn't you use a signed integer?

Reply:
no it doesn't need to be int but it needs to be signed because selecting "no column" is indicated by the value `-1`.
But I can make it an `int8_t` since the files won't be expected to have more than 128 columns.

Reply:
Ah ok, I thought `0` could take the special meaning of `-1`